src/font/lv_font.c

```diff
@@ -71,12 +71,23 @@ bool lv_font_get_glyph_dsc(const lv_font_t * font_p, lv_font_glyph_dsc_t * dsc_o
     bool found = false;
     while(f) {
         found = f->get_glyph_dsc(f, dsc_out, letter, letter_next);
+        dsc_out->resolved_font = f;
         if(found && !dsc_out->is_placeholder) {
```

Comment:
There can be 2 cases:
1. The glyph is found in a font -> great, we break and return;
2. Only a placeholder is found in a font -> We should save the font in which a placeholder was found, and if there is no better option use that.